apps/ledger-live-desktop/src/renderer/families/canton/PendingTransferProposals/index.tsx

```diff
@@ -49,24 +50,26 @@ const INSTRUCTION_TYPE_MAP: Record<TransferProposalAction, TransferInstructionTy
   withdraw: "withdraw-transfer-instruction",
 };
 
-const PendingTransferProposals: React.FC<Props> = ({ account }) => {
+const PendingTransferProposals: React.FC<Props> = ({ account, parentAccount }) => {
   const dispatch = useDispatch();
   const device = useSelector(getCurrentDevice);
   const unit = useAccountUnit(account);
   const sync = useBridgeSync();
   const [modal, setModal] = useState<Modal>({ isOpen: false, action: "accept", contractId: "" });
 
-  const cantonAccount = isCantonAccount(account) ? account : null;
-  const accountXpub = account.xpub ?? "";
+  const cantonAccount = isCantonAccount(parentAccount) ? parentAccount : null;
+  const accountXpub = parentAccount.xpub ?? "";
 
   const performTransferInstruction = useCantonAcceptOrRejectOffer({
-    currency: account.currency,
-    account,
+    currency: parentAccount.currency,
+    account: parentAccount,
     partyId: accountXpub,
   });
 
   const { groupedIncoming, groupedOutgoing, incomingCount, outgoingCount } = useMemo(() => {
-    const pendingTransferProposals = cantonAccount?.cantonResources?.pendingTransferProposals;
+    const pendingTransferProposals = (
+      account as { cantonResources?: { pendingTransferProposals?: RawTransferProposal[] } }
```

Comment:
That good, do you think we can do something like ?
```suggestion
      isCantonAccount(account) ? account.cantonResources.pendingTransferProposals : undefined;
```